slang.h

```diff
@@ -3374,6 +3375,11 @@ SLANG_API SlangResult spCompileRequest_getModule(
     SlangInt                translationUnitIndex,
     slang::IModule**        outModule);
 
+/** Get the `ISession` handle behind the `SlangCompileRequest`.
+*/
+SLANG_API SlangResult spCompileRequest_getSession(
+    SlangCompileRequest* request,
+    slang::ISession** outSession);
```

Comment:
I don't mind having this, but it should also be unnecessary because a client that is using the newer COM-like API would be calling `ISession::createCompileRequest`, so that there would be no question about which session a compile request belongs to.

Reply:
Kept this because our test infra uses the old API and I don't want to change that in this PR.